docs/howto/deploy/upgrade.md

```diff
@@ -136,86 +136,3 @@ To do so, add the following to your configuration (relevant **only** for 0.30.0)
 cataloger:
   type: rocks
 ```
-
```

Comment:
Can we instead add a note along these lines?

```suggestion
If you are using a version before 0.50.0, you must first perform the [previous upgrade to that version](https://docs.lakefs.io/v0.50/reference/upgrade.html#data-migration-for-version-v0500). {: note: .note-warning }

Reply:
👍